iceoryx2-ffi/ffi/README.md

```diff
@@ -0,0 +1,47 @@
+# Naming Convention
+
+- all constructs start with `iox2_`
```

Comment:
Can we cover those rules via clippy in the ffi implementation? Not for now but we could create an issue to explore this.

Reply:
Sounds good